Gemfile

```diff
@@ -67,15 +66,8 @@ explicitly_require_windows_gems = false
 puppet_gem_location = gem_type(puppetversion)
 # This is not a perfect answer to the version check
 if puppet_gem_location != :gem || puppetversion < '3.5.0'
-  is_x64 = Gem::Platform.local.cpu == 'x64'
-  if is_x64
-    platform(:x64_mingw) do
-      explicitly_require_windows_gems = true
-    end
-  else
-    platform (:mingw) do
-      explicitly_require_windows_gems = true
-    end
+  if Gem::Platform.local.os == 'mingw32'
```

Comment:
The intention was to always require gems on Windows all the time if we are overriding the Puppet Gem location or using a version less than 3.5.0. The old logic had a bug in that it would also include non-Windows plaforms if either of the other two concerns were met. The fix is https://github.com/ferventcoder/modulesync_configs/commit/518401d8f494b1b1f82754b48278e9bd13d2d315